script/update-archive-version.ts

```diff
@@ -50,51 +53,48 @@ async function updateArchiveAndContentVersions() {
   const osWebLoader = createOSWebLoader(`${ARCHIVE_URL}${REACT_APP_OS_WEB_API_URL}`);
 
   const currentBookLoader = makeUnifiedBookLoader(
-    createArchiveLoader(getArchiveUrl, {
-      archivePrefix: ARCHIVE_URL,
-    }),
-    osWebLoader
-  );
-
-  const getNewArchiveUrl = () => newArchiveUrl;
-
-  const newBookLoader = makeUnifiedBookLoader(
-    createArchiveLoader(getNewArchiveUrl, {
+    createArchiveLoader({
       archivePrefix: ARCHIVE_URL,
-      disablePerBookPinning: true,
```

Comment:
can you explain this? I feel like there was a case that led to us including it here.